### PR TITLE
Fix RTL4a.

### PR DESCRIPTION
**Pull request by tcard**
[Review on Reviewable](https://reviewable.io/reviews/ably/ably-ios/224)

### CODE REVIEW

**Comment by ricardopereira** [comment]
LGTM

File: ably-ios/ARTRealtimeChannel.m [hunk 1]
```diff
@@ -415,7 +415,7 @@ - (void)detach:(void (^)(ARTErrorInfo * _Nullable))cb {
         case ARTRealtimeChannelDetaching:
         case ARTRealtimeChannelDetached:
             [self.realtime.logger debug:__FILE__ line:__LINE__ message:@"can't detach when not attahed"];
-            if (cb) cb([ARTErrorInfo createWithCode:90000 message:@"Can't detach when not attahed"]);
+            if (cb) cb(nil);
```
Comment (mattheworiordan) [inline ably-ios/ARTRealtimeChannel.m]: Can you fix the typo in the already existing code on the line before, `attached` instead of `attahed`?

**Comment by mattheworiordan** [comment]
One minor request, but :+1:

**Comment by tcard** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[ably-ios/ARTRealtimeChannel.m, line 418 \[r1\]](https://reviewable.io:443/reviews/ably/ably-ios/224#-KApA26LJXdUMteihgg1-r1-418:-KApHb6lqDxNy8UznIW7:1661266018) ([raw file](https://github.com/ably/ably-ios/blob/cb6f6231ad481a6989cca978ebe13d9851f5ed59/ably-ios/ARTRealtimeChannel.m#L418)):*
Done at 7836bbd.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/ably/ably-ios/224)*

File: ably-ios/ARTRealtimeChannel.m (hunk 1 above)
Comment (tcard) [inline ably-ios/ARTRealtimeChannel.m]: Done at 7836bbd.

**Pull request merged**